File: 13. Java Basic Syntax - Lab/3 Integers Sum/StartUp.java
import java.util.Scanner;

public class StartUp {
    public static void main(String[] args) {
       Scanner scanner = new Scanner(System.in);

       int num1 = scanner.nextInt();
       int num2 = scanner.nextInt();
       int num3 = scanner.nextInt();
       
       if (!checkNums(num1, num2, num3) && !checkNums(num3, num1, num2) && !checkNums(num2, num3, num1)) {
            System.out.println("No");
       }       

       
    }
    
    static boolean checkNums(int num1, int num2, int sum) {
        if (num1 + num2 != sum) {
            return false;
        }
        if (num1 <= num2) {
            System.out.printf("%d + %d = %d%n", num1, num2, sum);
        }
        else {
            System.out.printf("%d + %d = %d%n", num2, num1, sum);
        }
        return true;
    }
}

File: 05. JavaScript Basic Syntax - Lab/Three Integers Sum/StartUp.js
function sum(input) {
    let nums = input[0].split(' ').map(Number);
    let num1 = nums[0];
    let num2 = nums[1];
    let num3 = nums[2];
    let result;
    
    if (num1 + num2 == num3){
        result = (`${Math.min(num1, num2)} + ${Math.max(num1, num2)} = ${num3}`);
    }
    else if (num2 + num3 == num1){
        result = (`${Math.min(num2, num3)} + ${Math.max(num2, num3)} = ${num1}`);
    }
    else if (num1 + num3 == num2){
        result = (`${Math.min(num1, num3)} + ${Math.max(num1, num3)} = ${num2}`);
    }
    else {
        result = ("No")
    }

    console.log (result);
}